openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/contributions_sociales/csg_crds.py

```diff
@@ -42,7 +42,10 @@ class crds(Variable):
     value_type = float
     entity = Individu
     label = 'Contributions au remboursement de la dette sociale'
-    reference = 'https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000190291/2021-10-06/'
+    reference = [
+        'Ordonnance n° 96-50 du 24 janvier 1996 relative au remboursement de la dette sociale, art. 14'
```

Comment:
Je pense qu'il faut mettre des virgules à la fin de chaîne de caractères de cette liste, car sinon elles sont prises comme une seule chaîne de caractère.
Et il vaut mieux utiliser des guillemets ("doubles quotes") dans les chaînes de caractères car les "simple quotes" peuvent être utilisés comme des apostrophes.

Reply:
@benjello Est-ce que tu penses à ça : 

```suggestion
        "Ordonnance n° 96-50 du 24 janvier 1996 relative au remboursement de la dette sociale, art. 14",
```